docs/DeveloperGuide.md

```diff
@@ -428,6 +429,43 @@ The `ArgumentMultimap` class is used to parse the user input and store the filte
 
 The `FilterDescriptor` takes in the filter criteria and returns a single predicate encompassing all the criteria on its `getFilters` method, so that this predicate can be used as an argument for the `updateFilteredPersonsList` method of the `Model` object, displaying a list of students that were filtered by this predicate.
 
+### Email Feature
+
+The email mechanism implements the following sequence for the method call execute("email").
+
+#### What is the email feature
+
+The email feature opens up a separate window containing the emails of the current list displayed in Tracey. The email feature allows a user to copy the email list to their clipboard to paste in their email platform of choice.
```

Comment:
Hi, maybe instead of "containing the emails of the current list", it will be better to say "containing the emails of the students found in the current list displayed"